src/bundle/Resources/public/scss/_navigation-menu.scss

```
@@ -0,0 +1,96 @@
+.ibexa-navigation-menu {
+    padding: calculateRem(25px);
+    font-size: $ibexa-text-font-size-medium;
+
+    &__primary {
+        height: calculateRem(48px);
+        padding: calculateRem(4px);
+        border-radius: $ibexa-border-radius;
+        background-color: $ibexa-color-light-300;
+        display: flex;
+    }
+
+    &__primary-item {
+        height: calculateRem(40px);
+        padding: calculateRem(10px);
+        border-radius: $ibexa-border-radius;
+        border: 0;
+        flex-basis: 100%;
+        text-align: center;
+
+        &--active {
+            background-color: $ibexa-color-white;
+            font-weight: 600;
+        }
+    }
+
+    &__secondary {
+        margin: calculateRem(24px) 0 0;
+        padding: calculateRem(24px) 0 0;
+        list-style: none;
+        display: none;
+        border-top: calculateRem(1px) solid $ibexa-color-light;
+
+        &--active {
+            display: block;
+        }
+    }
+
+    &__secondary-item-btn {
+        display: inline-flex;
+        width: 100%;
+        padding: calculateRem(11px) calculateRem(15px);
+        border: calculateRem(1px) solid transparent;
+        border-radius: $ibexa-border-radius;
+        text-align: left;
+
+        &::before {
+            content: '';
+            display: block;
+            width: calculateRem(8px);
+            height: calculateRem(8px);
+            margin: calculateRem(8px) calculateRem(16px) 0 0;
+            border-radius: 50%;
+            background-color: $ibexa-color-light;
+        }
+
+        &::after {
+            content: '*';
+            width: calculateRem(16px);
+            display: inline-block;
+            opacity: 0;
+            margin: calculateRem(4px) 0 0 calculateRem(4px);
+            color: $ibexa-color-danger;
+            font-size: $ibexa-text-font-size-small;
+        }
+
+        &:hover {
+            color: $ibexa-color-info;
+            font-weight: 600;
+
+            &:before {
```

here and in other places below:
```suggestion
            &::before {
```